test/bedrock_Azure_common_kv_test.go

```diff
@@ -0,0 +1,137 @@
+)
+
+func TestIT_Bedrock_AzureCommon_KV_Test(t *testing.T) {
+	t.Parallel()
+
+	//Generate common-infra resources for integration use with azure-simple environment
+	uniqueID := random.UniqueId()
+	k8sName := fmt.Sprintf("gTestk8s-%s", uniqueID)
+	addressSpace := "10.39.0.0/16"
+	kvName := k8sName + "-kv"
+	kvRG := kvName + "-rg"
+	location := os.Getenv("DATACENTER_LOCATION")
+	clientid := os.Getenv("ARM_CLIENT_ID")
+	subnetName := k8sName + "-subnet"
+	tenantid := os.Getenv("ARM_TENANT_ID")
+	vnetName := k8sName + "-vnet"
+
+	//Generate common-infra backend for tf.state files to be persisted in azure storage account
+	backendName:=		os.Getenv("ARM_BACKEND_STORAGE_NAME")
+	backendKey:=		os.Getenv("ARM_BACKEND_STORAGE_KEY")
+	backendContainer:=	os.Getenv("ARM_BACKEND_STORAGE_CONTAINER")
+	backendTfstatekey:=	k8sName +"-tfstatekey"
+
+	//Specify the test case folder and "-var" option mapping for the backend
+	common_backend_tfOptions := &terraform.Options{
+		BackendConfig: map[string]interface{}{
+			"storage_account_name": backendName,
+			"access_key":	backendKey,
+			"container_name":	backendContainer,
+			"key":	"common_"+backendTfstatekey,
+		},
+	}
+
+	//Specify the test case folder and "-var" option mapping
+	common_tfOptions := &terraform.Options{
+		TerraformDir: "../cluster/environments/azure-common-infra",
+		Upgrade:      true,
+		Vars: map[string]interface{}{
+			"address_space":	addressSpace,
+			"keyvault_name":	kvName,
+			"global_resource_group_name":	kvRG,
+			"global_resource_group_location":	location,
+			"service_principal_id":	clientid,
+			"subnet_name":	subnetName,
+			"subnet_prefix":	addressSpace,
+			"tenant_id":	tenantid,
+			"vnet_name":	vnetName,
+		},
+	}
+
+	//Terraform init, apply, output, and defer destroy for common-infra bedrock environment
+	defer terraform.Destroy(t, common_tfOptions)
+	terraform.Init(t, common_backend_tfOptions)
+	terraform.Apply(t, common_tfOptions)
+
+	//Obtain the vnet_subnet_id for the deployed vnet from the common-infra bedrock environment
+	commonInfra_subnetID := terraform.Output(t, common_tfOptions, "vnet_subnet_id")
+
+	// Generate azure single environment using resources generated from common-infra
+	dnsprefix := k8sName + "-dns"
+	clientsecret := os.Getenv("ARM_CLIENT_SECRET")
+	k8sRG := k8sName + "-rg"
+	publickey := os.Getenv("public_key")
+	sshkey := os.Getenv("ssh_key")
+
+	//Specify the test case folder and "-var" option mapping for the environment backend
+	k8s_backend_tfOptions := &terraform.Options{
+		BackendConfig: map[string]interface{}{
+			"storage_account_name": backendName,
+			"access_key":	backendKey,
+			"container_name":	backendContainer,
+			"key":	backendTfstatekey,
+		},
+	}
+
+	// Specify the test case folder and "-var" options
+	k8s_tfOptions := &terraform.Options{
+		TerraformDir: "../cluster/environments/azure-single-keyvault",
+		Upgrade:      true,
+		Vars: map[string]interface{}{
+			"address_space":			addressSpace,

```

Comment:
Can you run this file through `go format` so that the formatting is per the golang standard?